src/main/java/com/google/sps/data/Keywords.java

```diff
@@ -57,7 +57,9 @@
   public static String addKeywords(List<EntityAnnotation> blobAnalysis) throws IOException {
     List<String> labelList = new ArrayList<>();
     for (EntityAnnotation label : blobAnalysis) {
-      labelList.add(label.getDescription());
+      if (label.getScore() >= 0.75) {
```

Comment:
Maybe put a comment here explaining you only want to add high confidence labels.... (I assume thats what this is doing?)

Reply:
Yes.